docs/netlify/tools/convert.js

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
The file does not follow our formatting policy. Can I ask you to update the `docs:lint` script and add `netlify` directory to it? Then, after running `npm run docs:lint`, you should be able to see what needs to be fixed.
```diff
diff --git a/docs/package.json b/docs/package.json
index 779fdd193f..4a71d6adde 100755
--- a/docs/package.json
+++ b/docs/package.json
@@ -10,8 +10,8 @@
     "directory": "docs"
   },
   "scripts": {
-    "docs:lint": "eslint --ext .js,.vue,.mjs .vuepress content",
-    "docs:lint:fix": "eslint --ext .js,.vue,.mjs .vuepress content --fix",
+    "docs:lint": "eslint --ext .js,.vue,.mjs,.ts .vuepress content netlify",
+    "docs:lint:fix": "eslint --ext .js,.vue,.mjs,.ts .vuepress content netlify --fix",
     "docs:scripts:link-assets": "node .vuepress/tools/link-assets.mjs",
     "docs:scripts:generate-legacy-docs-versions": "node .vuepress/tools/generate-legacy-docs-versions.mjs",
     "docs:code-examples:generate-js": "bash .vuepress/tools/code-examples-generator.sh",
```

Reply:
Netlify directory included in the lint step. Added few exception, for very long regular expressions. Excetpion was added for the console.log, as it is supported by Netlify way of logging and these logs. Additionaly some exceptions were added for the Typescript types